st2chatops.env

```diff
@@ -33,7 +33,7 @@ export ST2_AUTH_USERNAME="${ST2_AUTH_USERNAME:-st2admin}"
 export ST2_AUTH_PASSWORD="${ST2_AUTH_PASSWORD:-testp}"
 
 # Public URL of StackStorm instance: used it to offer links to execution details in a chat.
-export ST2_WEBUI_URL=https://${ST2_HOSTNAME}
+export ST2_WEBUI_URL="${ST2_WEBUI_URL:-https://${ST2_HOSTNAME}}"
```

Comment:
Would be also nice to do the same for remaining env vars in this file:
`HUBOT_LOG_LEVEL`, `HUBOT_NAME` and `HUBOT_ALIAS`, and so we can close https://github.com/StackStorm/st2chatops/issues/50